src/app/home/components/Header.js

```diff
@@ -29,23 +33,34 @@ class Header extends React.Component {
       this.setState({
         position: 'fixed',
         animation: 'fadeStickyIn .6s ease-in-out forwards',
+        offset: (-55),
       });
     } else if (window.scrollY < 200) {
       this.setState({
         position: 'relative',
         animation: 'none',
+        offset: (-110),
       });
     }
   }
 
   render() {
+    const props = {
+      delay: 0, smooth: 'easeInOutQuart', duration: 1000, offset: (this.state.offset),
+    };
     return (
       <Box position={this.state.position} transition={this.state.animation}>
```

Comment:
Must use destructuring state assignment (coding style). Please add

const {
  offset, position, animation
} = this.state;

between "render() {" and "const props = {" and use offset, position, animation instead of this.state.offset, this.state.position, this.state.animation.